types/resourceDeleter.js

```diff
@@ -0,0 +1,78 @@
+/* @flow */
+
+/* Logger */
+export type LoggerOptions = {
+  error: (...args: Array<any>) => void,
+  info: (...args: Array<any>) => void,
+  warn: (...args: Array<any>) => void,
+  debug: (...args: Array<any>) => void,
+}
+
+/* Config */
+export type ApiConfigOptions = {
+  host: string,
+  projectKey: string,
+  credentials: {
+    clientId: string,
+    clientSecret: string,
+  },
+  scopes: Array<string>,
+  apiUrl?: string,
+}
+
+/* Client */
+// eslint-disable-next-line max-len
+export type MethodType =
+  | 'GET'
+  | 'POST'
+  | 'DELETE'
+  | 'HEAD'
+  | 'OPTIONS'
+  | 'PUT'
+  | 'PATCH'
+  | 'TRACE'
+
+export type ClientRequest = {
```

Comment:
and this one here: https://github.com/commercetools/nodejs/blob/0ded6ee95bc19a667f001f870f14f2d1c4aa30e9/types/sdk.js#L15